backend/sonar-project.properties

```
@@ -1,4 +1,4 @@
-sonar.projectKey=govuk-one-login_mobile-status-list-backend
+sonar.projectKey=mobile-status-list-backend
```

issue: Lets align with the repo name now. 
I can create the project if needed.

```suggestion
sonar.projectKey=crs-backend
```